peytonjo/overlook
# Feature/functionality

### What does this PR do?
- adds the functionality to user dashboard
- adds css styling to all search results 
- uses interpolation to add user info to dom

### How should this be tested?
- should be able to see all user past-present or future bookings
- should be able to filter rooms by date or by type

### Notes/ take aways:
- manager dashboard needs functionality shown through dom
- user should be able to add booking through room search
